test-requirements.txt

```diff
@@ -2,6 +2,6 @@
 # See LICENSE file for licensing details.
 
 black
-flake8==4.0.1
-flake8-copyright<0.3
+flake8<6
```

Comment:
Should this also be pinned like the others?

```txt
# Temporarily pinned to due flake8-copyright not yet supporting flake8 6.  This can be removed when
# [this fix](https://github.com/savoirfairelinux/flake8-copyright/pull/20) is merged.
flake8==4.0.1
```

Reply:
Agreed, switching to use 4.0.1